src/store/Accounts.ts

```diff
@@ -86,11 +86,11 @@ export class Accounts implements Module {
         updateCodes(state: AccountsState) {
           let second = new Date().getSeconds();
           if (UserSettings.items.offset) {
-            // prevent second from negative
-            second += Number(UserSettings.items.offset) + 60;
+            second += Number(UserSettings.items.offset);
           }
 
-          second = second % 60;
+          // prevent second from negative
+          second = (second < 0) ? (60 - (second * -1) % 60) : (second % 60);
```

Comment:
Just to make things a bit more obvious, lets make the OOO more obvious and remove the ternary opeator to make it clearer what the comment is referring to. 

```suggestion
          if (second < 0) {
            // Handle the situation where offset causes `second` to be negative. #1310 
            second = 60 - ((second * -1) % 60)
          } else {
            second = second % 60
          }
```

Reply:
hi and thanks for the suggestion, you can go ahead and make these changes to speed this up as I'm currently unavailable 🙂

Reply:
oh wait, I can accept the changes via the web interface, which I can't do in the mobile app 🙄